operators/src/source/gdal_source.rs

```diff
@@ -382,25 +394,93 @@ impl MetaData<GdalLoadingInfo, RasterResultDescriptor, RasterQueryRectangle>
 }
 
 impl GdalDatasetParameters {
+    /// Placeholders are surrounded by `%` and are replaced by formatted time value.
```

Comment:
I don't know, lots of template engines work this way, e.g. in TypeScript you use `${variable}`.
I hope we won't get many replacements since multiple replace calls on top of each other aren't a good solution.

06785c7